src/servises/formatString.ts

```diff
@@ -0,0 +1,3 @@
+export const formatString = (str: string): string => {
+  return str.toLowerCase().split(' ').join(' ');
```

Comment:
The current implementation of `formatString` does not modify the string as intended. If the goal is to normalize spaces, consider using a regular expression to replace multiple spaces with a single space, e.g., `str.replace(/\s+/g, ' ')`.